updated Symfony version to 3~

snsanich opened the pull request:
updated dependencies and fix tests.

gossi commented:
Thanks for upgrading. Can you adjust them a little bit, so people with older php version can still use this. For the required versions, for all symfony packages please use `~2|~3` and for phpunit `~4|~5`. Tests on travis should also pass then, I guess.

snsanich commented:
Unfortunately I found it is not possible to handle multiple versions in package.json. I updated .travis.yml and added there php7.0

php5.4,php5.5 are not supported. http://php.net/supported-versions.php

gossi commented:
Yep, it is possible. I realized I had a lot uncommitted changes locally. While I was giving them some love I also integrated your changes. The lib now works from php 5.4 up to 7 and hhvm. Thanks for your changes.
